frontend/src/metabase/browse/components/ModelsTable.tsx

```diff
@@ -125,13 +144,16 @@ const TBodyRow = ({ item }: { item: ModelResult }) => {
           <CollectionBreadcrumbsWithTooltip
             containerName={containerName}
             collection={item.collection}
+            // To avoid propagating the click event to the ModelTableRow
+            breadcrumbGroupProps={stopClickPropagation}
+            collectionsIconProps={stopClickPropagation}
```

Comment:
does this function really need to be passed here? can't you just set it in the components where it's used?